### PR TITLE
On stage icons - add new icons

### PR DESCRIPTION
**Pull request by bamai**
https://jira.wixpress.com/browse/DSM-3705

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-icons-common**
Commit: **fb35c280a4339a8f6a22f6c804aba9890b85b8e9**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-icons-common.pr_2364/

File: packages/wix-ui-icons-common/src/on-stage/general/metadata.ts [hunk 1]
```diff
@@ -1,27 +1,6 @@
...
-    tags: [],
-    sizes: { "20":"AdminSmall", "24":"Admin" },
-  },
-  {
-    title: "AdminFill",
-    category: "Social",
-    description: "",
-    tags: [],
-    sizes: { "20":"AdminFillSmall", "24":"AdminFill" },
-  },
```
Comment (jonathanadler) [inline packages/wix-ui-icons-common/src/on-stage/general/metadata.ts:24]: Why were these removed 🤔 ?

**Review by jonathanadler — APPROVED** [review]
Looks good 👍 
Just wondered about the changes in the metadata file
I see some definitions were moved or removed

File: packages/wix-ui-icons-common/src/on-stage/general/metadata.ts (hunk 1 above)
Comment (bamai) [reply packages/wix-ui-icons-common/src/on-stage/general/metadata.ts:24]: They are not supposed to be removed 🤔 I think I moved them down in the file to where all icons of Social category are...
In general I changed the order of the metadata file a bit, so that all icons from the same category will be together (and they are sorted in alphabetical order within the category)

Comment (jonathanadler) [reply packages/wix-ui-icons-common/src/on-stage/general/metadata.ts:24]: ohhh, I see
Cool 👌

**Comment by EgoziE** [comment]
wix-ui-icons-common was moved to another location.
Please see readme page for more information.
https://github.com/wix/wix-ui/blob/master/packages/wix-ui-icons-common/README.md
